src/qibolab/platform.py

```diff
@@ -95,6 +96,8 @@ class Platform:
 
     couplers: CouplerMap = field(default_factory=dict)
     """Dictionary mapping coupler names to :class:`qibolab.couplers.Coupler` objects."""
+    kernel_folder: Optional[Path] = None
+    """Folder where each qubit kernels are stored"""
```

Comment:
This could be useful but I believe it is never assigned, neither in the tests, nor in the example https://github.com/qiboteam/qibolab_platforms_qrc/pull/90. So maybe we can just drop for now to simplify?